index.html

```diff
@@ -8,5 +8,6 @@
   </head>
   <body>
     <div id="root"></div>
+    <script type="module" src="main.jsx"></script>
```

Comment:
Do you need the hardcoded JS link?

Reply:
From what I have reviewed, it is required to run it in developer mode. Anyway, vite removes that reference and re-references it in the app build